test_login.py

```diff
@@ -231,7 +231,7 @@ class LoginTestCase(unittest.TestCase):
     ''' Tests for results of the login_user function '''
 
     def setUp(self):
-        self.app = Flask(__name__)
+        self.app = Flask(__name__, subdomain_matching=True)
```

Comment:
Do we need a test for `False` as well?

Reply:
When `subdomain_matching = False` (the default),  all of the tested subdomain routes will 404 so no login behavior can actually be tested for subdomain URLs. (Looks like the flag itself was added as an explicit setting in Flask 1.0 per https://stackoverflow.com/a/62265895).

My thinking was to save some boilerplate by just setting `subdomain_matching=True` here. However, if we are concerned about wanting to keep `subdomain_matching=False` as the default false setting for the other non-subdomain login tests, I could pull the subdomain tests out into a separate class though.